scripts/in_container/check_connectivity.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```

Comment:
:heart: this one to be searated :)

Reply:
Yeah this can be used multiple places so separated :)